packages/combobox/src/Combobox.ts

```diff
@@ -416,8 +434,12 @@ export class Combobox extends Textfield {
                 ?readonly=${this.readonly}
             />
             ${when(
```

Comment:
I feel you might remove the when directive calls whenever possible. This is a function call and it will create some overheads for simpler if and else condition, but its a thought and not a blocker!

Reply:
This is a convenience wrapper around a ternary expression that makes it a little nicer to write an inline conditional without an else statement.